docs/UserGuide.adoc

```diff
@@ -295,6 +295,63 @@ To end step-by-step instruction of cook-mode. +
 Format: +
 `end`
 
+=== Health-planner Command
+
+==== Add plan: `add`
+
+Adds a health plan under the user. +
+Required parameters are GOAL and DURATION +
+GOAL specifies a string input of "GAIN", "LOSS" or "MAINTAIN" +
+DURATION specifies an integer input of time in days desired by user to achieve said GOAL +
```

Comment:
For parameter specifications do put them in sections within "****" under Format.
e.g.
```
==== Search ingredient

Find ingredients whose name contains any of the given keywords.
Format: +
`find KEYWORD…`

****
* Keywords are case insensitive
* Only the name is searched
* Match only full words
****
```